pom.xml

```
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>de.kekru.struktogrammeditor</groupId>
 	<artifactId>struktogrammeditor</artifactId>
-	<version>1.7.3-SNAPSHOT</version>
+	<version>1.7.4</version>

```

Please change to

```suggestion
	<version>1.7.4-SNAPSHOT</version>
```

Removing the "SNAPSHOT" is only when I do the release. The target master branch should always contain SNAPSHOT